cuda/apps/src/kinect_dediscretize.cpp

```diff
@@ -79,8 +79,6 @@ class SimpleKinectTool
 
       std::function<void (const openni_wrapper::Image::Ptr& image, const openni_wrapper::DepthImage::Ptr& depth_image, float)> f = std::bind (&SimpleKinectTool::cloud_cb_, this, _1, _2, _3);
 
-      boost::signals2::connection c = interface->registerCallback (f);
```

Comment:
Can't remove a 'registerCallback` since it is actually used

Reply:
```suggestion
      interface->registerCallback (f);
```

Reply:
Thank you @kunaltyagi